books/free-programming-books-subjects.md

```diff
@@ -826,6 +826,9 @@ Kerridge (PDF) (email address *requested*, not required)
 ### Quantum Computing
 
 * [Introduction to Classical and Quantum Computing](https://www.thomaswong.net/introduction-to-classical-and-quantum-computing-1e3p.pdf) - Thomas G. Wong (PDF)
+* [Quantum Computation and Quantum Infromation](https://profmcruz.wordpress.com/wp-content/uploads/2017/08/quantum-computation-and-quantum-information-nielsen-chuang.pdf) - Michael A Nielsen
```

Comment:
This appears not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines.